### PR TITLE
Feature/ted 236

### PR DESCRIPTION
**Pull request by CaptainOfHacks**

### CODE REVIEW

File: ted_sws/mapping_suite_processor/services/conceptual_mapping_files_injection.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          the resources in resources_folder_path will be copied to output_resource_folder_path.
+    :param conceptual_mappings_file_path:
+    :param resources_folder_path:
+    :param output_resources_folder_path:
+    :return:
+    """
+    resources_df = pd.read_excel(conceptual_mappings_file_path,
+                                         sheet_name=CONCEPTUAL_MAPPINGS_RESOURCES_SHEET_NAME)
+    resource_file_names = list(resources_df[FILE_NAME_KEY].values)
+    for resource_file_name in resource_file_names:
```
Comment (costezki) [inline ted_sws/mapping_suite_processor/services/conceptual_mapping_files_injection.py:25]: What happens with the resources that do not exist?
Spelling errors for example? These cases must be carefully treated .

File: tests/unit/mapping_suite_processor/polygon.ipynb [hunk 2]
```diff
@@ -1,97 +0,0 @@
-{
- "cells": [
```
Comment (costezki) [inline tests/unit/mapping_suite_processor/polygon.ipynb:2]: Do we need this file?

**Review by costezki — APPROVED** [review]
Well done

File: tests/unit/mapping_suite_processor/polygon.ipynb (hunk 2 above)
Comment (CaptainOfHacks) [reply tests/unit/mapping_suite_processor/polygon.ipynb:2]: we don't need

File: ted_sws/mapping_suite_processor/services/conceptual_mapping_files_injection.py (hunk 1 above)
Comment (CaptainOfHacks) [reply ted_sws/mapping_suite_processor/services/conceptual_mapping_files_injection.py:25]: shutil will raise error

**Pull request merged**